server/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #[macro_use]
 extern crate diesel;
+use deadpool_lapin::{Manager, Pool};
```

Comment:
A bit of a nit but can these just be full scope pathed, we have so many Manager and Pool types that this could get confusing very fast